pages/[user]/[type].tsx

```diff
@@ -76,25 +81,33 @@ export default function Type(props): Type {
     setSelectedDate(dayjs().month(selectedMonth).date(day));
   };
 
+  const checkAvailableDays = (day) => {
+    if (selectedMonth > minMonth && selectedMonth < maxMonth) {
+      return true
+    }
+    else if (selectedMonth > minMonth && selectedMonth === maxMonth) {
+      return day <= maxDay
+    }
+    else if (selectedMonth === minMonth && selectedMonth < maxMonth) {
+      return day >= minDay
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689307)
```suggestion
      return day >= minDay;
```